server/routes/api/conversations.js

```diff
@@ -22,7 +22,7 @@ router.get("/", async (req, res, next) => {
       attributes: ["id"],
       order: [[Message, "createdAt", "DESC"]],
       include: [
-        { model: Message, order: ["createdAt", "DESC"] },
+        { model: Message, order: ["createdAt", "ASC"] },
```

Comment:
When running the app, were you able to verify this fixed/changed the ordering? Since changing line 23 like you did before also changes the order of the conversations so we'll have to find another way to solve it.

Reply:
Thanks for your correction. I updated by querying using updatedAt for conversations and ordering messages in asc order. I am still familiarizing myself with Sequelize.